subsys/bluetooth/mesh/Kconfig

```
@@ -620,6 +641,21 @@ config BT_MESH_RELAY_RETRANSMIT_INTERVAL
 	  messages, in milliseconds. Can be changed through runtime
 	  configuration.
 
+config BT_MESH_RELAY_BUF_COUNT
+	int "Number of advertising buffers for relayed messages"
+	default 32
```

I would suggest to keep this default low as we have found out. 
```suggestion
	default 7
```

---

Do we really have to change this value, I guess that it was set to 32 when first selected.
The users that need a lower value can change this in their configuration, and for future we can change it in main branch. I am worried about if this could have some side effect without us knowing it.

---

Do we really have to change this value, I guess that it was set to 32 when first selected.
The users that need a lower value can change this in their configuration, and for future we can change it in main branch. I am worried about if this could have some side effect without us knowing it.

---

Ok. We should update this upstream main branch.